src/lib.rs

```diff
@@ -104,19 +104,20 @@ impl From<HeliumNetId> for proto::HeliumNetId {
 pub struct OrgResponse {
     pub org: Org,
     pub net_id: hex_field::HexNetID,
-    pub devaddr_constraints: Vec<DevaddrConstraint>,
+    pub devaddr_constraints: Vec<DevaddrSubnet>,
 }
 
 impl From<proto::OrgResV1> for OrgResponse {
     fn from(res: proto::OrgResV1) -> Self {
+        let constraints: Vec<DevaddrConstraint> = res
+            .devaddr_constraints
+            .into_iter()
+            .map(|d| d.into())
```

Comment:
Can you do this conversion to a subnet in a single map like with `.map(|d| d.into::<DevAddrConstraint>().to_subnet()` and avoid the second iteration?